src/components/buttons/async-button.cjsx

```diff
@@ -55,7 +54,7 @@ module.exports = React.createClass
 
     if isFailed or isTimedout
       stateClass = 'is-failed'
-      return <failedState {...failedProps}/>
+      return React.createElement(failedState, failedProps)
```

Comment:
come to think of it, i think we could just capitalize the prop altogether as we are not using the customizable `FailedState` feature of this button anywhere yet.  It would look like this change https://github.com/openstax/tutor-js/commit/7e9bcf666c19e48d4e8941d6fe10e5d6c7175bd2